example/hyprland.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
minor peeves: can you add a comment here and in the other service that this is an example systemd service with a tiny desctiption